variables.tf

```diff
@@ -127,6 +127,12 @@ variable "database_version" {
   }
 }
 
+variable "database_vm_size" {
+  description = "Specifies the vm size to be allocated for the database"
+  type        = string
+  default     = "db-n1-standard-2"
```

Comment:
I'm inclined to making the default to `db-n1-standard-2` at the least.

Reply:
Could we do that in a seperate PR

Reply:
acutally nvm, we will just set standard-1 on the managed installs